libs/ballot-interpreter-nh-next/src/interpret.rs

```diff
@@ -213,6 +224,17 @@ pub fn load_ballot_page_image(image_path: &Path) -> core::result::Result<LoadedB
     })
 }
 
+#[time]
+fn make_normalized_image_buffer(image: ImageBuffer<Luma<u8>, Vec<u8>>) -> NormalizedImageBuffer {
```

Comment:
I think you can just use this alias:
```suggestion
fn make_normalized_image_buffer(image: GrayImage) -> NormalizedImageBuffer {
```